crates/hotshot/Cargo.toml

```diff
@@ -8,7 +8,7 @@ version = { workspace = true }
 rust-version = { workspace = true }
 
 [features]
-default = ["docs", "doc-images"]
+default = ["docs", "doc-images", "dependency-tasks"]
```

Comment:
Not a strong opinion but could we just remove the feature-gating entirely? Even without the feature, I think if we ever want to revert in the future we'd only have to change which consensus tasks we spawn in `hotshot/src/tasks/mod.rs`

Reply:
I think the issue would be that we can't keep running the legacy tests while it's still deployed in prod and we might need to hotfix it